### DISCUSSION TITLE
“ in the normal mode does not show the details of registers anymore

### DISCUSSION THREAD

**Discussion by xyecoding** [LunarVim/LunarVim#3068] [Q&A]
When press `"` in the normal mode, the details of registers will be showed before. However, after I update `lunarvim` and all the plungins, it is not be showed anymore. I do not sure this is a function of `lunarvim` or `which-key`. How can I get this function again.

**Comment by LostInTheLogs** [comment]
```lua
lvim.builtin.which_key.setup.plugins.registers = true
```
